pkgs/applications/misc/qlandkartegt/51-garmin.rules

```diff
@@ -0,0 +1 @@
+ATTR{idVendor}=="091e", ATTR{idProduct}=="0003", GROUP="uucp", MODE="660"
```

Comment:
Which Garmin devices does this entry cover? It looks like that entry should be rather part of a local OS configuration.

Reply:
http://www.the-sz.com/products/usbid/index.php?v=0x091E&p=0003&n=:
> 0x091E
0x0003
Garmin International
GPS (various models)

Reply:
Just looking at the USB database: a lot of other Garmin models have different product IDs (the product ID for my 64st is not even listed there). It would prefer taking this file out of the PR (rather make it part of your local config).

Reply:
OK, removed.